### ISSUE TITLE
Not an issue, it just sucks this is dead

### ISSUE THREAD

**Issue by Sonico98** [innocenat/typingmania#1]
Amazing game, would've loved if this got more attention :(

**Comment by innocenat** [comment]
Sorry about that.

I think the current code still works as intended, I think. But a lot of new technology has become mainstream since then and a rewrite would be better at this point. I am working on them, but I don't have much free time nowadays.

**Comment by Sonico98** [comment]
> Sorry about that.
> 
> I think the current code still works as intended, I think. But a lot of new technology has become mainstream since then and a rewrite would be better at this point. I am working on them, but I don't have much free time nowadays.

Don't say sorry! It's not your fault. Good luck with the rewrite

**Comment by innocenat** [comment]
Hello.

I just pushed the rewrite version into new branch (which I made default). The documentation and tooling around it is still lacking and I am still working on it.

The new main feature is that it now supports video, either locally, or embeded via YouTube. 

Keeping this issue open until I finished all documentation and tooling.

Example screen playing LiSA's Homura via YouTube: https://i.imgur.com/1U142Ek.jpg
